bin/migrate

```diff
@@ -3,5 +3,5 @@ set -e
 
 python manage.py migrate
 python manage.py migrate_clickhouse
-python manage.py run_async_migrations
-python manage.py sync_replicated_schema
+python manage.py run_async_migrations --check
+python manage.py sync_replicated_schema
```

Comment:
@macobo should this be done after async migrations check?

Reply:
Well, yes, that's where I put it.

Reply:
Why is is there not before async migrations?
Is there any scenario, where the `sync_replicated_schema` influences async migrations `is_required` check?

Reply:
This discussion is not in scope for your PR and it's correct as it is.